backend/src/main/java/de/neuefische/backend/controller/ControllerTasks.java

```diff
@@ -28,4 +28,9 @@ public TaskModel addNewTask(@RequestBody TaskModel taskModel) {
     public void deleteById(@PathVariable String id) {
         serviceTasks.deleteById(id);
     }
+
+    @PutMapping({"/{id}"})
+    public TaskModel editTask(@PathVariable String id, @RequestBody TaskModel taskModel) {
+        return serviceTasks.editTask(taskModel);
```

Comment:
is there a reason you're not using a DTO here?

Reply:
i dont think i needed it here.